server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/GetMailboxesMethod.java

```diff
@@ -88,83 +89,82 @@ public GetMailboxesMethod(MailboxManager mailboxManager, QuotaRootResolver quota
     }
 
     @Override
-    public Stream<JmapResponse> processToStream(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession) {
+    public Flux<JmapResponse> process(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession) {
         Preconditions.checkArgument(request instanceof GetMailboxesRequest);
         GetMailboxesRequest mailboxesRequest = (GetMailboxesRequest) request;
 
+        return metricFactory.runPublishingTimerMetricLogP99(JMAP_PREFIX + METHOD_NAME.getName(),
+            () -> process(methodCallId, mailboxSession, mailboxesRequest)
+            .subscriberContext(context("GET_MAILBOXES", mdc(mailboxesRequest))));
```

Comment:
is GET_MAILBOXES duplicated in `mdc` method?

Reply:
We could have a constant, yes, however the context key is just a blind marker. IMO context could be generating a random string on the caller behalf.